controllers/dashboardController.js

```diff
@@ -0,0 +1,63 @@
+
+const jwt = require("jsonwebtoken")
+
+const config = require("../config")
+const conferences = require("../lib/conferences")
+const { decrypt } = require("../lib/crypto")
+
+module.exports.get = async (req, res) => {
```

Comment:
la convention jusqu'ici c'etait plutot
```suggestion
module.exports.getDashboard = async (req, res) => {
```